tests/models/decoder_only/vision_language/test_models.py

```diff
@@ -291,6 +291,15 @@
     #     vllm_output_post_proc=model_utils.phi3v_vllm_to_hf_output,
     #     num_logprobs=10,
     # ),
+    "pixtral_hf": VLMTestInfo(
+        models=["nm-testing/pixtral-12b-FP8-dynamic"],
+        test_type=(VLMTestType.IMAGE, VLMTestType.MULTI_IMAGE),
+        prompt_formatter=lambda img_prompt: f"<s>[INST]{img_prompt}[/INST]",
```

Comment:
```suggestion
        prompt_formatter=lambda img_prompt: f"<s>[INST]\n{img_prompt}[/INST]",
```

Reply:
From the HF example, the newline is actually supposed to be between the prompt and the images i.e.
```
PROMPT = "<s>[INST]Describe the images.\n[IMG][IMG][IMG][IMG][/INST]"
```

https://github.com/vllm-project/vllm/blob/77f7ef29088fef854421239e7c41df6b11bc4b5b/examples/offline_inference_vision_language.py#L295

I wasn't sure how to do this with this structure, but it wasn't important to get matching vLLM vs HF

Reply:
Oh, I see now. You can use the `img_idx_to_prompt` argument for this.

Reply:
Yeah I did use `img_idx_to_prompt=lambda idx: "[IMG]",` and that worked thankfully. Luckily the newline isn't crucial for correctness, so that feature can wait